src/main/java/dukeApp/files/Storage.java

```diff
@@ -0,0 +1,81 @@
+package dukeApp.files;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    protected String filePath;
+    ArrayList<Task> arrList = new ArrayList<>();
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Load date from file and store in an ArrayList
+     * @return task list
+     * @throws FileNotFoundException if the file does not exist
+     */
+    public ArrayList<Task> load() throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        Task t;
+        boolean isDone;
+
+        while (s.hasNext()) {
```

Comment:
You can abstract the isDone = ... and arrList.add(t) statements out of the if else loop since it is used in every case.